tst/extreme/translat.tst

```diff
@@ -0,0 +1,52 @@
+#############################################################################
+##
+#W  extreme/translat.tst
+#Y  Copyright (C) 2016-17                                          Finn Smith
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+gap> START_TEST("Semigroups package: extreme/translat.tst");
+gap> LoadPackage("semigroups", false);;
+
+#
+gap> SEMIGROUPS.StartTest();
+
+#T# RZMS Translational Hull
```

Comment:
In the intervening years we've got rid of these `#T#` and `#E#` tags, so we may as well not reintroduce them now. So please just use `#`. 🙂  Same with the standard test file.